### PR TITLE
Closes AgileVentures/MetPlus_tracker#434

### PR DESCRIPTION
**Pull request by arun1595**
- Add policies and policies specs
- Update cuke scenarios and rspec tests

### CODE REVIEW

File: features/company_person.feature [hunk 1]
```diff
@@ -160,8 +160,8 @@ Feature: Company Person
     And I should not see selections of "Feature Inc." addresses
     And I select "12 Main Street Detroit, Michigan 02034" in select list "Address"
     Then I click "Update Company person" button
-    And I should be on the Company person 'jane@widgets.com' show page
-    And I should see "12 Main Street Detroit, Michigan 02034"
```
Comment (joaopapereira) [inline features/company_person.feature]: I think this check should be kept to ensure the changes were done correctly

File: spec/controllers/company_people_controller_spec.rb [hunk 2]
```diff
@@ -4,8 +4,9 @@
 RSpec.describe CompanyPeopleController, type: :controller do
```
Comment (joaopapereira) [inline spec/controllers/company_people_controller_spec.rb]: We need negative tests to ensure all the people that is not supposed to enter do not enter.
Take a look at spec/controllers/skills_controller_spec.rb

**Comment by joaopapereira** [comment]
@arun1595 did you had time to review this pr?

**Comment by arun1595** [comment]
@joaopapereira I am working on it. I will add the rspec tests and update the PR.

**Comment by joaopapereira** [comment]
@arun1595 Any news on this PR?

**Comment by arun1595** [comment]
@joaopapereira I have added some negative tests but then noticed that we're missing rspec tests for some actions. Can we move that part to a new story?

**Comment by joaopapereira** [comment]
@arun1595 if they are not related with this yes, please create a story in waffle and mark it as a chore so we can remember

**Comment by joaopapereira** [comment]
The Issues opened with the missing tests is AgileVentures/MetPlus_tracker#469

**Comment by joaopapereira** [comment]
Going to merge

**Pull request merged**